src/lib/webpack/webpack-base-config.js

```diff
@@ -220,7 +220,7 @@ export default (env) => {
 		// produce HTML & CSS:
 		addPlugins([
 			new ExtractTextPlugin({
-				filename: 'style.css',
+				filename: isProd ? "style.[contenthash:5].css" : "style.css",
```

Comment:
I love the idea on client, not so on server side. I'd much prefer to have static file on server for critical css inlining.

I think we should pass an argument to `webpack-base-config` from `webpack-client-config` and `webpack-server-config`